README.md

```diff
@@ -83,4 +83,9 @@ docker-compose -p greedybear up
 
 # shut down the application
 docker-compose -p greedybear down
-```
\ No newline at end of file
+```
+Note that GreedyBear *needs* a running instance of ElasticSearch to function.
```

Comment:
"Note that GreedyBear *needs* a running instance of ElasticSearch to function. This should be the ElasticSearch of a TPoT."